client-participation/polis.config.template.js

```diff
@@ -6,13 +6,14 @@ module.exports = {
     "^192\\.168\\.1\\.140$",
     "^pol\\.is",
     ".+\\.pol\\.is$",
+    ".+\\.polisdemo\\.tech$",
```

Comment:
Seems odd to include one persons' dev preference, which is a real domain? Maybe whitelist all `.+\\.local$` instead and advise ppl use that if they're messing with their `/etc/hosts`? (Maybe I'm missing something!)

Reply:
I bought the polisdemo.tech domain and hope to use it as a publicly accessable demo site. it doesn't need to be here yet, but I think it could be neat to make use of it in the near future.

Reply:
removed. will revisit this in the future.

Reply:
Demo site is a great idea, +1

Reply:
Demo site is great idea! I have a question, but I'll bug you in chat or on call :)